workshops/geometric_pattern/README.md

```diff
@@ -455,9 +476,17 @@ If your page has frozen, you've probably introduced an infinite loop. Refresh yo
 
 Save to see the changes!
 
+Now that we've done more work, let's commit again:
+
+- `git add --all`
+- `git commit -m "Reverse direction of circles in geometric_pattern"`
+- `git push`
+
 ## Part IV: Adding Color
 
-Black and white is nice, but how about some color? p5.js provides several helpful functions, including: [`color()`](http://p5js.org/reference/#p5/color), [`fill()`](http://p5js.org/reference/#p5/fill), and [`stroke()`](http://p5js.org/reference/#p5/stroke) that help modify the graphics.
+Black and white is nice, but how about some color? We've previously used CSS to change colors, but this is a special case -- CSS can't modify anything inside of a canvas.
```

Comment:
It might be worth it to reiterate here that p5.js operates on a `canvas` element, I'm not sure how many people will remember otherwise.